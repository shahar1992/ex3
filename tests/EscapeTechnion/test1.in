## General public tests

company add roomsinc@gmail.com 13
room add roomsinc@gmail.com 1 100 3 06-18 7
escaper add escmaster@gmail.com 13 8

escaper order escmaster@gmail.com 13 1 0-10 3
report day

escaper recommend escmaster@gmail.com 3
report day

report best

escaper remove escmaster@gmail.com
room remove 13 1
company remove roomsinc@gmail.com
