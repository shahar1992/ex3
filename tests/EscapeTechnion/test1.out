Day #0: 1 events in total
The order was made by escmaster@gmail.com, from the faculty of Computer Science, who is skilled at 8
	It was made for room 1, of roomsinc@gmail.com from Computer Science, for the time 10:00
	The room is best for a skill level of 7. 3 people came for the total price of 225
That is all for day #0!

Day #1: 1 events in total
The order was made by escmaster@gmail.com, from the faculty of Computer Science, who is skilled at 8
	It was made for room 1, of roomsinc@gmail.com from Computer Science, for the time 0:00
	The room is best for a skill level of 7. 3 people came for the total price of 225
That is all for day #1!

The system has 18 faculties and in 2 days, they have made a revenue of 450
One of the best faculties - Computer Science, has made a revenue of 450 so far!
One of the best faculties - Civil and Environmental Engineering, has made a revenue of 0 so far!
One of the best faculties - Mechanical Engineering, has made a revenue of 0 so far!

